string-manipulation-3/src/title-case.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-unused-vars */
+
+const titleCase = title => {
+  const minorWords = ['in', 'of', 'the', 'for'];
```

Comment:
Try to generalize your solution to account for all rules in the problem statement. Our goal is to make a solution that works on any title, not just the example inputs.